src/Uno.Templates/content/unoapp/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CsWinRTAotOptimizerEnabled>false</CsWinRTAotOptimizerEnabled>
```

Comment:
Thanks. We won't this this in the templates directly at this time, we'll adjust that in the SDK.